internal/sdkv2provider/resource_cloudflare_workers_queue.go

```diff
@@ -0,0 +1,131 @@
+package sdkv2provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/pkg/errors"
+)
+
+func resourceCloudflareWorkersQueue() *schema.Resource {
+	return &schema.Resource{
+		Schema:        resourceCloudflareWorkersQueueSchema(),
+		CreateContext: resourceCloudflareWorkersQueueCreate,
+		ReadContext:   resourceCloudflareWorkersQueueRead,
+		UpdateContext: resourceCloudflareWorkersQueueUpdate,
+		DeleteContext: resourceCloudflareWorkersQueueDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceCloudflareWorkersQueueImport,
+		},
+		Description: "Provides the ability to manage Cloudflare Workers Queue features.",
+	}
+}
+
+func resourceCloudflareWorkersQueueCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	client := meta.(*cloudflare.API)
+
+	accountID := d.Get("account_id").(string)
+	if accountID == "" {
+		accountID = client.AccountID
```

Comment:
`client.AccountID` is deprecated. instead, we should enforce `account_id` and rely on that value explicitly.